docs/manual/manual_stow.rst

```diff
@@ -219,12 +219,61 @@ Refer to the :ref:`timeout_behavior` section for more information on how to conf
 
 Holding Position
 ==================
-Once the module reaches the stow position, it can either continue actively trying to hold that position or it can coast the motor and let it spin freely. This 
-behavior is controlled by the *Hold Stow* parameter, as mentioned in the :ref:`stow_parameter_summary` section above.
+Once the module reaches the stow position, it can transition to one of several different behaviors. Which behavior is controlled by the *Hold Stow* parameter, as mentioned in the :ref:`stow_parameter_summary` section above.
+The possible behaviors are:
+
+* **Coast When Stowed**: The module will release the stow and coast, allowing itself to spin freely. It will not attempt to hold its stow position.
+* **Hold Stow Angle**: The module will attempt to actively hold its stow position using the position controller. This can be useful if you need the module to maintain its stowed position when external forces are acting on it, such as the wind. 
+* **Brake When Stowed**: The module will release the stow and brake. The module is not actively attempting to hold its stow position, but brake mode does provide more resistance to moving than coast.
+* **Low Power Hold**: The module will attempt to hold its stow target position, but it will swap between brake mode and actively using the position controller to try and move to the stow position depending on how far it is from the stow position. This is described in more detail below.
+
+Low Power Hold
+****************
+The Low Power Hold behavior is more complex than the other stow ending behaviors and has several configurations of its own. This section describes the Low Power Hold behavior in detail.
+
+Low Power Hold reduces power consumption by switching between brake mode and actively attempting to hold the target position with the position controller depending on how far the module is from the target position.
+Brake mode consumes relatively little power, but only provides marginal resistance to moving out of position. Actively holding the position with the position controller forces the module back to its target position, but it consumes
+significantly more power in doing so. So, in Low Power Hold, the module goes into brake mode when it is close to its target position, and switches back to actively holding the target position when it is too far away from the target.
+Once the module is again close enough to its target, the module transitions back into brake mode. This trades off precisely maintaining the target position at all times for reduced power consumption.

```

Comment:
This behavior continues, thus precisely maintaining ...

Reply:
Actually my point here is that we aren't precisely maintaining the position anymore. I'm trying to tell people that we are trading off precise positioning, because we aren't constantly pushing ourselves to stay in perfect position, to get less power consumption. We aren't doing such precise position maintenance anymore, using low power hold means giving that up to get the low power. I can re-phrase this if that's not clear

Reply:
I changed the wording and moved some things around to make my meaning clearer